### ISSUE TITLE
Multi-Track Rendering

### ISSUE THREAD

**Issue by Danielku15** [CoderLine/alphaTab#35]
It would be great if multiple tracks could be rendered in a single canvas. 

Possible implementation: 

1. Add a new renderMultiple(tracks:Array<Track>) method. 
https://github.com/CoderLine/alphaTab/blob/master/src/alphatab/rendering/ScoreRenderer.hx#L82

2. Extend the stave creation to create staves for all tracks
https://github.com/CoderLine/alphaTab/blob/master/src/alphatab/rendering/layout/ScoreLayout.hx#L61

3. Extend this method to hint the track which the stave is for:
addStave(trackIndex:Int, stave:Stave) https://github.com/CoderLine/alphaTab/blob/master/src/alphatab/rendering/staves/StaveGroup.hx#L94

4. Replace the addBar with a more multi-track variant. 
maybe something like addBar(score:Score, barIndex:Int). 
https://github.com/CoderLine/alphaTab/blob/master/src/alphatab/rendering/staves/StaveGroup.hx

5. Extend the accolade rendering to render the track names at the beginning. 
https://github.com/CoderLine/alphaTab/blob/master/src/alphatab/rendering/staves/StaveGroup.hx#L150

Basically it will be an additional grouping of the StaveGroup. The BarRenderers will handle a multi track layouting already.
